sirepo/auth_role_moderation.py

```diff
@@ -22,11 +22,81 @@
 
 _cfg = None
 
+
+@sirepo.api_perm.require_adm
+def api_admModerate():
+
+    def _send_moderation_status_email(info):
+        approved_denied_subject = f'{info.app_name} Access Request '
```

Comment:
I think this could be a module constant:

```py
_STATUS_TO_SUBJECT = PKDict(
    approve={} Access Request Approved',
    clarify='Sirepo {}: Additional Info?',
    deny={} Access Request Denied',
)
```

The denormalization is fine in this case, because the text strings are so close together. There's only two uses, too. It simplifies the code, and there's no great harm if there's a spelling error or whatever.

Reply:
ad61fc813610afe4d4c57cbe0e4322c503d6ac6c